gpii/node_modules/flowManager/src/Capture.js

```diff
@@ -0,0 +1,229 @@
+/**
+ * GPII snapshot Handler
```

Comment:
"snapshot Handler" -> "Capture Component"?

Please fix the year at line 4 too.

Reply:
Updated the header